main.go

```diff
@@ -387,14 +391,21 @@ func checkService(opts *Opts, client *k8s.Clientset) error {
 	}
 
 	logger.Infow("Target identified", "IP", serviceIP, "Port", servicePort)
-	go runForwarder(serviceIP, int(servicePort), opts)
+	go runForwarder(serviceIP, servicePort, opts)
 	targetService = service
 	return nil
 }
 
-func runForwarder(serviceIP string, servicePort int, opts *Opts) {
+func runForwarder(serviceIP, servicePort string, opts *Opts) {
 	for {
 		logger.Info("Starting forwarder")
+		fluentTargetIP := serviceIP
+
+		if opts.KonnectivityUDSSocket != "" {
+			// TODO set up the tunnel
```

Comment:
There is a TODO?

Reply:
Oops, that was a leftover old comment. TODO is now done. :smile: